addons/mail/static/src/js/form_renderer.js

```diff
@@ -91,9 +91,7 @@ FormRenderer.include({
      */
     async _forceMountChatterComponent() {
         this._chatterComponent.__owl__.isMounted = false;
-        if (this._chatterContainerTarget) {
-            await this._mountChatterComponent();
-        }
```

Comment:
Removed as `_forceMountChatterComponent` is only called if `this._chatterComponent` is defined and if `this._chatterComponent` is defined then `this_chatterContainerTarget` is defined too